# Force allowing REAL(2) and REAL(3) in flang tests.
# REAL(2) and REAL(3) are explicitly disabled in EPI and flang tests
# that use them fail without this patch
--- flang/test/lit.cfg.py.ORIG	2026-07-15 17:08:53.798220219 +0200
+++ flang/test/lit.cfg.py	2026-07-15 17:14:57.764595936 +0200
@@ -146,12 +146,13 @@
     ToolSubst(
         "%flang",
         command=FindTool("flang"),
+        extra_args=["-Xflang", "-fforce-real-2", "-Xflang", "-fforce-real-3"],  
         unresolved="fatal",
     ),
     ToolSubst(
         "%flang_fc1",
         command=FindTool("flang"),
-        extra_args=["-fc1"],
+        extra_args=["-fc1", "-fforce-real-2", "-fforce-real-3"],
         unresolved="fatal",
     ),
 ]
